.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: python
+
+matrix:
+  include:
+    - python: 3.7
+      dist: xenial
+      sudo: true
+
+script: pytest tests/unit/*
```

Comment:
could we also add linting checks? and move to appveyor as well.

Reply:
There isn't anything to lint, currently. I would definitely consider adding this as we test individual templates, though.